dockstore-webservice/src/main/resources/migrations.1.10.0.xml

```diff
@@ -77,4 +77,22 @@
     <changeSet author="nolwarre (generated)" id="notificationsCharLimit">
         <modifyDataType columnName="message" newDataType="varchar(1024)" tableName="notification"/>
     </changeSet>
+    <changeSet author="gluu (generated)" id="1601569350016-13">
+        <createTable tableName="collection_entry_version">
+            <column autoIncrement="true" name="id" type="SERIAL">
```

Comment:
Why does this join table have its own id (possible inspiration from `user_entry` or `organization_user` which don't have their own id.

Reply:
Other tables don't need an ID because they use a composite key (based on all their non-nullable) columns, since version id is null-able, it isn't possible.

Reply:
interesting